### PR TITLE
Add documentation on Screen Sharing for mobile app

### PR DESCRIPTION
**Pull request by mironiasty**
## Description

Add info about screen sharing and background run of app (for React Native)

### CODE REVIEW

File: guide/react-native/background.mdx [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+sidebar_position: 6
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Streaming from background
+
+On Android, it is possible to continue streaming when app is in background. Unfortunately this functionality is not available on iOS (due to Apple limitations)
```
Comment (MiloszFilimowski) [inline guide/react-native/background.mdx:10]: **nitpick:** Are you sure this is true? I think it's possible to achieve using CallKit (at least audio, this is what google meets does), however our app does not support it.

File: guide/react-native/background.mdx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      ...
+    }
+  }
+}
+```
+
+  </TabItem>
+  <TabItem value="rn" label="Bare workflow">
+
+You need to modify `AndroidManifest.xml` file and add below service:
```
Comment (MiloszFilimowski) [inline guide/react-native/background.mdx:43]: **issue:** If we focus right now on only expo configuration, the above plugin is enough. If we want to support not expo/manual config this is too little to make it work. I'd remove it for now as it only creates confusion.

File: guide/react-native/screensharing.mdx [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+
+# Screen sharing
+
+Our SDK also allow to stream content of mobile device screen.
+
+## Installation
+
+### Android
+
+On Android devices, screen share feature is available right after SDK is installed in app. No further actions are required.
```
Comment (MiloszFilimowski) [inline guide/react-native/screensharing.mdx:16]: **issue:** This is not true. Screen sharing won't work on Android >=24 if you don't enable `enableForegroundService`, as we need a foreground service for that to work.

File: guide/react-native/screensharing.mdx [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+      ...
+    }
+  }
+}
+```
+
+  </TabItem>
+  <TabItem value="rn" label="Bare workflow">
+
+Configuring screen sharing on iOS is a little complicated.
```
Comment (MiloszFilimowski) [inline guide/react-native/screensharing.mdx:51]: **suggestion:** I'd mention that below config is not needed if you use the above plugin as it does all that.

**Review by MiloszFilimowski — APPROVED** [review]

File: guide/react-native/background.mdx (hunk 1 above)
Comment (mironiasty) [reply guide/react-native/background.mdx:10]: Here is more investigation done by Angelika. We might work on this in future, to make it work somehow.

**Pull request merged**